net-libs/openpgm/openpgm-5.3.128_p20200831.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Comment:
Python 3.6 possible to keep?

Reply:
It would work just fine, but py3.6 support is going out in Gentoo if I understood correctly.